arch/arm/Kconfig

```diff
@@ -6,6 +6,21 @@
 if ARCH_ARM
 comment "ARM Options"
 
+config ARCH_CORTEXM
```

Comment:
where we check these new flags in the code?

Reply:
It should be used soon, let us mark is as draft first